components/sendgrid/actions/get-a-global-supression/get-a-global-supression.js

```diff
@@ -0,0 +1,35 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-get-a-global-supression",
+  name: "Get A Global Supression",
+  description: "Gets a global supression.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    email: {
+      type: "string",
+      label: "Email",
+      description:
+        "The email address of the global suppression you want to retrieve.",
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```